### PR TITLE
prevent restricted usernames

### PR DESCRIPTION
**Pull request by ebarthur**

### CODE REVIEW

File: client/app/routes/create-account.tsx [hunk 1]
```diff
@@ -36,6 +37,19 @@ export const action = async ({ request }: ActionFunctionArgs) => {
 
 	const { password, email, username } = await request.json();
 
+	for (const restrictedUsername of restrictedUsernames) {
+		if (username === restrictedUsername) {
+			return json(
+				{
+					type: "conflict",
+					field: "username",
+					message: "Username is not allowed.",
```
Comment (blackmann) [inline client/app/routes/create-account.tsx]: Let's just say "Username already exists"

**Review by blackmann — CHANGES_REQUESTED** [review]
Beautiful.

Please address the comment then I'll merge

File: client/app/lib/restrict-usernames.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	"notgr",
+	"compa",
+	"compa.so",
+	"admin",
+	"sysadmin",
+	"info",
+	"username",
+	"user",
+	"welcome",
+	"moderator",
```
Comment (blackmann) [inline client/app/lib/restrict-usernames.ts:11]: Let's add hello?

File: client/app/lib/restrict-usernames.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export const restrictedUsernames = [
+	"gr",
```
Comment (blackmann) [inline client/app/lib/restrict-usernames.ts]: You can't enter two letters anyway

**Review by ebarthur — COMMENTED** [review]
All comments resolved. You can merge mow. 

Thank you.

**Review by blackmann — APPROVED** [review]

**Pull request merged**